backend/api/Areas/Tools/Models/Import/ImportPropertyModel.cs

```diff
@@ -3,31 +3,33 @@ namespace Pims.Api.Areas.Tools.Models.Import
     public class ImportPropertyModel
```

Comment:
With the Excel Converter Tool, we can now expect better formatted data.  Updated model to match.